test/scenarios/applicationmonitoring/without_csi.go

```diff
@@ -0,0 +1,128 @@
+//go:build e2e
+
+package applicationmonitoring
+
+import (
+	"context"
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects/address"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/namespace"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/sampleapps"
+	sample "github.com/Dynatrace/dynatrace-operator/test/helpers/sampleapps/base"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/steps/assess"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/steps/teardown"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/tenant"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+const (
+	sampleAppNamespace = "appmon-sample"
+)
+
+func withoutCSIDriver(t *testing.T) features.Feature {
+	builder := features.New("application monitoring without csi driver enabled")
+	secretConfig := tenant.GetSingleTenantSecret(t)
+	defaultDynakubeName := "dynakube"
+	dynakubeBuilder := dynakube.NewBuilder().
+		WithDefaultObjectMeta().
+		Name(defaultDynakubeName).
+		ApiUrl(secretConfig.ApiUrl).
+		NamespaceSelector(metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"inject": defaultDynakubeName,
+			},
+		}).
+		ApplicationMonitoring(&dynatracev1beta1.ApplicationMonitoringSpec{
+			UseCSIDriver: address.Of(false),
+		})
+
+	appOnlyDynakube := dynakubeBuilder.Build()
+
+	dynakubeBuilder = dynakube.NewBuilder().
+		WithDefaultObjectMeta().
+		Name("appmon-sample").
+		WithDynakubeNamespaceSelector().
+		ApiUrl(secretConfig.ApiUrl).
+		ApplicationMonitoring(&dynatracev1beta1.ApplicationMonitoringSpec{
+			UseCSIDriver: address.Of(false),
+		})
```

Comment:
why do you need to use 2 dynakubes ?

you can have multiple sample-apps for the same-dynakube